src/index.html

```diff
@@ -22,7 +22,7 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+<html spellcheck="false">
```

Comment:
@RaymondLim Your solution works only for static pages, where all input/textarea fields are already existent on pageload.